pandas/io/formats/style.py

```diff
@@ -965,10 +977,10 @@ def hide_columns(self, subset):
     # -----------------------------------------------------------------------
 
     @staticmethod
-    def _highlight_null(v, null_color):
+    def _highlight_null(v, null_color) -> str:
```

Comment:
looks like null_color is implicitly treated as a str.  should it be typed?

Reply:
mypy ok with it being typed as ```str```